Update intuse registry text (#459)
ietf-rats-wg/eat: draft-ietf-rats-eat.md

## Intended Use Registry {#int-use-registry}

IANA is requested to create a new registry titled "EAT Intended Uses" in a new registry group 
called "Remote Attestation Procedures (RATS)."
The registry uses the "Expert Review Required" registration procedure {{RFC8126}}.

Guidelines for experts:

> * Each intended use should be clearly described so a user of can know what it means.

Reviewer: typo (mine)

Need to add "it"... "as user of it can know"

: This is a text paragraph or more that sufficiently defines what the intended use means. It may also be a reference to another document.

> These 5 values are the initial content of the registry are listed below.  Note that 0 will not be assigned for the CBOR value, and the maximum CBOR value for assignment is 255.  

Reviewer 2: ```suggestion
The following 5 values represent the initial content of the registry.  Note that 0 will not be assigned for the CBOR value, and the maximum CBOR value for assignment is 255.  
```
